tests/unittests/operators/delegated_tests.py

```
@@ -826,14 +826,31 @@ def test_rerun_with_renamed_dataset(self, get_op_mock):
     def test_rerun_child_do_fail(self, mock_get_operator):
```

Ruff (0.14.10)

Unused method argument: `mock_get_operator`

(ARG002)